src/components/calcite-action-group/calcite-action-group.e2e.ts

```diff
@@ -1,10 +1,26 @@
 import { accessible, hidden, renders } from "../../tests/commonTests";
+import { newE2EPage } from "@stencil/core/testing";
 
 describe("calcite-action-group", () => {
   it("renders", async () => renders("calcite-action-group"));
 
   it("honors hidden attribute", async () => hidden("calcite-action-group"));
 
+  it("defaults", async () => {
+    const page = await newE2EPage();
+
+    await page.setContent(`
+      <calcite-action-pad>
+        <calcite-action-group>
+          <calcite-action text="hello"></calcite-action>
+        </calcite-action-group>
+      </calcite-action-pad>
+      `);
+    const element = await page.find("calcite-action-group");
+    expect(element.getAttribute("layout")).toBe("vertical");
```

Comment:
There's a [test helper](https://github.com/Esri/calcite-app-components/blob/master/src/tests/commonTests.ts#L93) you can use for defaults.

Reply:
Cool. I will use taht.